requesting signoff for localization subsystem.md

Hawk652 opened the pull request:
requesting signoff for localization subsystem

Hawk652 committed:
Create Localization Subsystem.md

requesting signoff for localization subsystem

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

sgmandody42 committed:
Update Localization Subsystem.md

Hawk652 committed:
Update Localization Subsystem.md

updated the circuit schematic to properly represent the system

Hawk652 committed:
Update Localization Subsystem.md

updated the BOM to more accurately represent the subsystem's components

gabeakim committed:
Create Localization Subsystem.md

Added details on the system's connections in the analysis section

Hawk652 committed:
Update Localization Subsystem.md

added a figure of the current for the subsystem

Hawk652 committed:
Update Localization Subsystem.md

updated the BOM to better represent the components of the localization subsystem

Hawk652 committed:
Update Localization Subsystem.md

DrVanNeste reviewed (approved):
I approve this module, though I am not sure exactly how to approve it haha. I am clicking the approve button here in the comments.

Hawk652 committed:
Update Localization Subsystem.md

Removed the safety constraint from the list of constraints. The constraints will be moved to the safety subsystem.

gabeakim committed:
added explanation of localization accuracy

sgmandody42 committed:
Update Localization Subsystem.md

inserting code flowchart

gabeakim committed:
Changed lidar section and fixed reference order

sgmandody42 committed:
Update Localization Subsystem.md

fixed the flowchart

bouncyemma committed:
Update Localization Subsystem.md

added schematic that shows the connections of the different components

bouncyemma committed:
Update Localization Subsystem.md

added the separate schematic of the wireless beacon

bouncyemma committed:
Update Localization Subsystem.md

added sentence to describe wireless beacon

jawilkin2 committed:
Update flowchart description

Hawk652 committed:
Update Localization Subsystem.md

gabeakim committed:
Housing unit changes

Added use case for the housing unit. Updated the bill of materials.

Hawk652 commented on Documentation/Signoffs/Localization Subsystem.md:
```diff
@@ -0,0 +1,105 @@
+
+$P(x_k, m|z0_k, u0_k, x_0)$
+
+Accuracy of SLAM is dependant on the quality of the LiDar sensor but can have error as low as "5 cm given that LiDar sensor depth accuracy already has 2-3 cm error" [8]. The type of SLAM algorithm used will also impact the accuracy of the model. This information means the constraints of being within 15 cm of the path and 1m of the destination, when used in conjunction with the UWB, can be fulfill ...
```
This does not say why the 15 cm or how it will be achieved.

Say the 15 cm is based on the literature that is referenced. Using the literature, explain how the 15 cm was achieved in the literature so it can be replicated for the project.

jawilkin2 committed:
LiDar is not a correct capitalization

jawilkin2 committed:
Update image reference locations

jawilkin2 committed:
Fix image reference

jawilkin2 committed:
Add reference to image of ESP 32 USB DW3000

Hawk652 committed:
Update Localization Subsystem.md

Hawk652 committed:
Update Localization Subsystem.md

fixed the reference section

gabeakim committed:
Updated details regarding UWB beacons
